pygame_gui/core/ui_element.py

```diff
@@ -861,3 +861,13 @@ def hide(self):
 
         self.hovered = False
         self.hover_time = 0.0
+
+    def get_appropriate_state(self):
```

Comment:
I think this function name is too non-specific for what it does right now. If it gets more functionality later then would be a good time to change the name.

Also, I think it is a good candidate for being a protected function (with the single leading underscore in the name) since it's only really intended to be called by `on_unhovered()`. Perhaps call it `_get_enabled_state_name()` for now?